components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -84,6 +86,12 @@ exec $HAB_ROOT_PATH/bin/hab pkg exec core/hab-plan-build hab-plan-build \$*
 EOF
   $bb chmod $v 755 $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/build
 
+  $bb cat <<TAIL_SUP > $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/tail_sup
```

Comment:
Yeah I think that ideally we would have a `hab sup start -d` and the `-d` would indicate daemonized and it would emit logs in a sane way to a known location with rotation. Then `hab sup log` would consume those logs. I'm tempted just to do that here, but should just leave it for a separate pr.